src/main/java/io/github/microcks/testcontainers/MicrocksContainer.java

```diff
@@ -561,6 +564,115 @@ public void downloadAsSecondaryRemoteArtifact(String remoteArtifactUrl) throws A
       downloadArtifact(remoteArtifactUrl, false);
    }
 
+   /**
+    * Verifies that given Service has been invoked at least one time, for the current invocations' date.
+    *
+    * @param serviceName    mandatory
+    * @param serviceVersion mandatory
+    * @return false if the given service was not found, or if the daily invocation's count is zero. Else, returns true if the daily invocations' count for the given service is at least one.
+    */
+   public boolean verify(String serviceName, String serviceVersion) {
+      return this.doVerify(serviceName, serviceVersion, null);
+   }
```

Comment:
Could you add `static` version of these methods with first parameters being the `microcksContainerHttpEndpoint` like in https://github.com/microcks/microcks-testcontainers-java/blob/main/src/main/java/io/github/microcks/testcontainers/MicrocksContainer.java#L303 ? This help making these methods available to the `microcks-quarkus` DevService where we don't have access to the container object itself but just to its exposed URL. Thanks!

Reply:
Done